frappe/utils/safe_exec.py

```diff
@@ -125,6 +125,12 @@ def get_safe_globals():
 	out._iter_unpack_sequence_ = RestrictedPython.Guards.guarded_iter_unpack_sequence
 	out.sorted = sorted
 
+	installed_apps = frappe._dict()
+	all_apps = frappe.get_installed_apps()
+	for app in all_apps:
+		installed_apps[app] = __import__(app)
```

Comment:
this puts everything that we restricted back into the unrestricted list. this is not what should be done here.